### PR TITLE
Disable camera scenes setting

### PR DESCRIPTION
**Pull request by SpontanCombust**
- added menu option to disable camera scenes on key events
- moved camera playing specifically to seperate functions for cutscenes
- added disable camera scenes entry to strings

### CODE REVIEW

File: src/settings.ws [hunk 1]
```diff
@@ -46,6 +46,9 @@ class RE_Settings {
 
   public var foottracks_ratio: int;
 
+  // Spontan edit
```
Comment (SpontanCombust) [inline src/settings.ws:49]: you can delete that

File: mod-menu.xml [hunk 2]
```diff
@@ -5466,6 +5466,7 @@
         <PresetsArray>
             <Preset id="0" displayName="rer_defaults" >
                 <Entry varId="RERfoottracksRatio"		        value="100"	/>
+                <Entry varId="RERcameraCutscenesDisabled"		value="0"	/>
```
Comment (Aelto) [inline mod-menu.xml:5469]: @SpontanCombust i think there is a mistake here, below the varId is `RERcameraScenesDisabled` when here it's `RERcameraCutscenesDisabled`. It means the preset won't work.

Comment (Aelto) [reply mod-menu.xml:5469]: Make sure it's the same ID you use in the settings too 👍

Comment (SpontanCombust) [reply mod-menu.xml:5469]: Yeah, you're right. I forgot to change it for the preset.

**Review by Aelto — CHANGES_REQUESTED** [review]
There is a mistake with the preset in the mod menu i think. I made a comment about it

**Review by Aelto — APPROVED** [review]

**Comment by Aelto** [comment]
Alright, the code is perfect i'm merging it. Thank you very much!

**Pull request merged**